_scripts/adb-reverse.sh

```diff
@@ -6,3 +6,12 @@ adb reverse tcp:9010 tcp:9010 # OAuth server
 adb reverse tcp:1111 tcp:1111 # Profile server
 adb reverse tcp:1111 tcp:1112 # Profile server "CDN"
 adb reverse tcp:5000 tcp:5000 # Sync server
+adb reverse tcp:3000 tcp:3000 # React APP
+adb reverse tcp:3031 tcp:3031 # payments server
+adb reverse tcp:8290 tcp:8290 # Graphql API
+adb reverse tcp:8001 tcp:8001 # cirrus (experimenter)
+adb reverse tcp:9292 tcp:9292 # Fortress
+adb reverse tcp:8080 tcp:8080 # 123done
+adb reverse tcp:10139 tcp:10139 # 321done
+adb reverse tcp:5050 tcp:5050 # browserid-verifier
+adb reverse tcp:8002 tcp:8002 # pushbo
```

Comment:
```suggestion
adb reverse tcp:8002 tcp:8002 # pushbox
```

Typo from my copy-pasta, sorry.